Exec/science/xrb_mixed/problem_initialize.H

```diff
@@ -0,0 +1,43 @@
+#ifndef problem_initialize_H
+#define problem_initialize_H
+
+#include <prob_parameters.H>
+#include <eos.H>
+#include <model_parser.H>
+
+#ifdef NSE_THERMO
+#include <nse.H>
+#endif
+
+AMREX_INLINE
+void problem_initialize ()
+{
+
+    const Geometry& dgeom = DefaultGeometry();
+
+    const Real* problo = dgeom.ProbLo();
+    const Real* probhi = dgeom.ProbHi();
+
+    if (problem::num_vortices > problem::max_num_vortices) {
+        amrex::Error("num_vortices too large, please increase max_num_vortices and the size of xloc_vortices");
+    }
+
+    // Read initial model
+    read_model_file(problem::model_name);
+
+    if (ParallelDescriptor::IOProcessor()) {
+        for (int i = 0; i < model::npts; i++) {
+            std::cout << i << " " << model::profile(0).r(i) << " " << model::profile(0).state(i,model::idens) << std::endl;
+        }
+    }
+
+    // velocity perturbation stuff
+
+    Real offset = (probhi[0] - problo[0]) / problem::num_vortices;
```

Comment:
Is the change here with respect to the Fortran intentional?

Reply:
no, it wasn't.  I just copied this vortex initialization from another problem.  And looking at it now, I'm not sure which we'd prefer.  I think this new method makes more sense with periodic BCs, because then they are evenly spaced.